cmd/ucp_client_bundles.go

```diff
@@ -0,0 +1,148 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/thebsdbox/diver/pkg/ucp"
+)
+
+var username, bundleID, labelName string
+
+func init() {
+
+	// The username flag is required right now until I can find a nicer way of getting the currently logged-in
+	// user to the backend function without a lot of duplication
+	ucpCliBundleList.Flags().StringVar(&username, "username", "", "Username to list bundle for (required)")
+	ucpCliBundleList.MarkFlagRequired("username")
+
+	ucpCliBundleDelete.Flags().StringVar(&username, "username", "", "Username to delete bundle for (required)")
+	ucpCliBundleDelete.MarkFlagRequired("username")
+	ucpCliBundleDelete.Flags().StringVar(&bundleID, "id", "", "ID of the bundle to delete (required)")
+	ucpCliBundleDelete.MarkFlagRequired("id")
+
+	ucpCliBundleRename.Flags().StringVar(&username, "username", "", "Username to rename bundle for (required)")
+	ucpCliBundleRename.MarkFlagRequired("username")
+	ucpCliBundleRename.Flags().StringVar(&bundleID, "id", "", "ID of the bundle to rename (required)")
+	ucpCliBundleRename.MarkFlagRequired("id")
+	ucpCliBundleRename.Flags().StringVar(&labelName, "name", "", "New name for the bundle (required)")
+	ucpCliBundleRename.MarkFlagRequired("name")
+
+	ucpCliBundle.AddCommand(ucpCliBundleDownload)
+	ucpCliBundle.AddCommand(ucpCliBundleList)
+
+	if !DiverRO {
```

Comment:
The DiverRO things will be moved to using go `build tags` so that I can supply a read-only version that can only be used to retrieve information and not potentially damage existing systems. Perhaps replace or improve `dsinfo` or something like that.